### PR TITLE
Assignment1 - Johanne McClenahan and Alexa Darrington

### PR DESCRIPTION
**Pull request by JohanneJayde**
@alyxadarr - Alexa's GitHub ID

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -5,23 +5,38 @@ public class Program
...
             }
         }
+        Console.WriteLine("Congratulations " + PlayerName + "!");
         Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
     }
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return ((float)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```
Comment (SaaWest) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:31]: exactly what I got. Good job!

**Review by SaaWest — COMMENTED** [review]
Pull Request title includes "Assignment 1" in the title ✔
    Partner has also made a commit ✔
    Pull Request targets Assignment 1 branch ✔
    Issue 1: Application no longer crashes ✔
    Issue 2: Unit test properly passes ✔

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs [hunk 2]
```diff
@@ -70,6 +70,22 @@ public void GetFilePath_ReturnsFileThatExists()
...
     }
+    [TestMethod]
+    [DataRow("Juan", "Juan")]
+    [DataRow("Mr. Roger", "Mr. Roger")]
+    public void GetPlayerName_ReturnsExpectedName(string inputName, string expectedName)
+    {
+
+        // Arrange
+        using StringReader sr = new StringReader(inputName);
+        Console.SetIn(sr);
```
Comment (BenjaminMichaelis) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:81]: Clever!

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 3]
```diff
@@ -5,23 +5,38 @@ public class Program
...
+    }
+    public static string GetPlayerName()
+    {
+        string playerName = " ";
+        Console.WriteLine("Please enter your name: ");
```
Comment (BenjaminMichaelis) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:36]:
```suggestion
        Console.Write("Please enter your name: ");
```
nit: I think the formatting of entering on the same line as this text may be slightly cleaner

**Review by BenjaminMichaelis — COMMENTED** [review]
Great job!

**Pull request closed without merging**